src/utils/fetcher/index.ts

```diff
@@ -1,6 +1,12 @@
-import { Configuration, MoviesApi } from "@/__generated__/api";
+{
+  /*
+  We need change absolute import path to general import, because when run test webpack can't detect folder /__generated__/
+*/
```

Comment:
Ini jangan lupa dikasih `TODO:` ya, biar nanti nyarinya gampang

```suggestion
  We need change absolute import path to general import, because when run test webpack can't detect folder /__generated__/
  
  TODO: fix webpack and use absolute path
*/
```